dtls.c

```diff
@@ -4026,6 +4029,7 @@ handle_ccs(dtls_context_t *ctx, dtls_peer_t *peer,
     }
   }
 
+  peer->handshake_params->hs_state.read_epoch++;
```

Comment:
We might want to add `assert(... read_epoch) > 0` below this line as a reminder that at some point, overly large epochs need to be addressed.

Reply:
OK.